shawerma/models.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+import django.utils.timezone
+from django.db.models import Max,Count,Avg,Sum
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class User(AbstractUser):
+    pass
+
+#edited
+class MenuItem(models.Model):
+    name = models.CharField(max_length=100)
+    price = models.DecimalField(max_digits=8, decimal_places=2)
+    created_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.name
+
+class Order(models.Model):
+    user=models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
what is this user? is it the User requesting this Order? or the worker who is working in the stop and writing the order?